### ISSUE TITLE
Annotations location are not precise/changing after saving

### ISSUE THREAD

**Issue by AIVinci** [harshalitalele/openseadragon-circular-annotation#2]
The annotation place is not which we annotate but the little far away. this is not helpful in medical field as i am trying to use. can you please correct that. thank you for developing the resource. 
Please try to fix the above bugs.
1. Annotations after clicking save move a bit away.
2. 1st time rectangle annotation is not saved.

**Comment by harshalitalele** [comment]
Yes I am aware of this issue. I can look into this after this week. I would appreciate if you can help with anything until then.
